x-pack/plugins/enterprise_search/server/lib/analytics/add_analytics_collection.ts

```diff
@@ -60,9 +60,9 @@ const createDataView = async (
   return dataViewsService.createAndSave(
     {
       allowNoIndex: true,
-      namespaces: [getDataStreamName(analyticsCollection.id)],
+      title: getDataStreamName(analyticsCollection.id),
+      name: getDataViewName(analyticsCollection.id),
```

Comment:
The data view was broken.
`title` is used for the index pattern and `name` for the data view name.